src/Development/IDE/Core/Rules.hs

```diff
@@ -251,53 +251,89 @@ getLocatedImportsRule =
             Nothing -> pure (concat diags, Nothing)
             Just pkgImports -> pure (concat diags, Just (moduleImports, Set.fromList $ concat pkgImports))
 
+type RawDepM a = StateT (RawDependencyInformation, IntMap ArtifactsLocation) Action a
+
+execRawDepM :: Monad m => StateT (RawDependencyInformation, IntMap a1) m a2 -> m (RawDependencyInformation, IntMap a1)
+execRawDepM act = execStateT act
+                    (RawDependencyInformation IntMap.empty emptyPathIdMap IntMap.empty
```

Comment:
Please don’t make indentation depend on identifier length. That just results in unnecessarily large diffs in the future.

Reply:
I think you should merge this PR and fix small things like this after merging.

Reply:
Making a new PR has more overhead than fixing it here. I can’t just push to master.

Reply:
Happy to push to your branch if you don’t want to fix it yourself.

Reply:
pushed the fix, I hope you don’t mind.